### PR TITLE
BI-11883: Add company name to basket delivery page

### PR DESCRIPTION
**Pull request by VaughanJackson123**

### CODE REVIEW

File: src/model/error.messages.ts [hunk 1]
```diff
@@ -10,8 +10,10 @@ export const ORDERS_DETAILS_FIRST_NAME_EMPTY: string = "Enter your first name";
 export const ORDERS_DETAILS_LAST_NAME_EMPTY: string = "Enter your last name";
 export const ORDER_DETAILS_FIRST_NAME_MAX_LENGTH: string = "First name must be 32 characters or fewer";
 export const ORDER_DETAILS_LAST_NAME_MAX_LENGTH: string = "Last name must be 32 characters or fewer";
+export const ORDER_DETAILS_COMPANY_NAME_NAME_MAX_LENGTH: string = "Company name must be 32 characters or fewer";
```
Comment (hwilliams1) [inline src/model/error.messages.ts:13]: Are we sure about this error? Company Names can be up to 160 Characters long in the db

File: src/views/delivery-details.html [hunk 2]
```diff
@@ -67,6 +67,17 @@
...
 
+        {{ govukInput({
+          label: {
+            text: "Company name"
+          },
+        classes: "govuk-!-width-two-thirds",
+        id: "companyName",
+        name: "companyName",
+        value: companyName,
+        errorMessage: companyNameError
```
Comment (hwilliams1) [inline src/views/delivery-details.html:78]: shouldn't the indentation be one step in on these lines? so they line up with the label contents to show as being part of the govukInput?

**Review by hwilliams1 — COMMENTED** [review]
I have added a couple of comments, the rest looks ok to me

File: src/views/delivery-details.html (hunk 2 above)
Comment (VaughanJackson123) [reply src/views/delivery-details.html:78]: I'll take a look at that indentation. 👍

File: src/model/error.messages.ts (hunk 1 above)
Comment (VaughanJackson123) [reply src/model/error.messages.ts:13]: I asked Frank about this. After consultations with Barri (and Rob?), he asked me to make it 32 like the other cases for now. Stories will be created in the future to make it longer throughout.

File: src/views/delivery-details.html (hunk 2 above)
Comment (VaughanJackson123) [reply src/views/delivery-details.html]: Hopefully fixed indentation issues in this template now.

**Review by hwilliams1 — APPROVED** [review]

**Pull request merged**